release-notes/swan-lake-release-notes/swan-lake-2202.2.0-release-note.md

```diff
@@ -141,11 +141,30 @@ digraph "org/package:0.1.0" {
 
 ### Breaking changes
 
+### Developer tools updates
+
+#### New features
+
+##### OpenAPI Tool
+- Added support for allowing resource method capability for the client generation command. 
+  Introduced `--client-methods <resource|remote>` option to select the client method type,  which can be `resource` or `remote`.
+  - `$ bal openapi -i <OpenAPI contract> --client-methods resource`
+  - `$ bal openapi -i <OpenAPI contract> --mode client --client-methods resource`
+
+#### Improvements
+
+##### OpenAPI Tool
+- Added support to validate the values that have been assigned to the generated Ballerina types concerning the given OpenAPI schema
+  validation using the API provided by the Ballerina constraint package. This validation is available for `int`, `float`, `number`, `string`, and `array`.
```

Comment:
```suggestion
  validation using the API provided by the Ballerina `constraint` package. This validation is available for `int`, `float`, `number`, `string`, and `array`.
```
Why do we have a line break here?

Reply:
I think it is a mistake, hadn't an intention to add a line break. will remove it